pkgs/by-name/st/sturmflut/package.nix

```
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, pkg-config
+, imagemagick
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation {
+  pname = "sturmflut";
+  version = "git";
```

According to the [package naming conventions](https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming) the version should be as follows:

```suggestion
  version = "0-unstable-2023-04-25";
```